### DISCUSSION TITLE
Every rooms gets the same error

### DISCUSSION THREAD

**Discussion by pelebel** [DigiLive/mushroom-strategy#144] [Q&A]
Newbie here, gog this error for every room:

 Error loading the view strategy:

TypeError: undefined is not an object (evaluating 'm(this,a,"f",l).domains')

 Any idea where to start? And yes I did try everything proposed in the wiki

**Comment by DigiLive** [comment]
Please file a [bug report](https://github.com/AalianKhan/mushroom-strategy/issues/new?assignees=&labels=bug&projects=&template=bug_report.md&title=).

**Reply by pelebel** [reply]
I tried to reproduce the error and I found out I did not install a prerequisite. After installing it, I had to recreate the dashboard from scratch and apply the strategy. So, case closed!
